### ISSUE TITLE
Bug with packets

### ISSUE THREAD

**Issue by matyasvane** [HibiscusMC/HMCLeaves#6]
```
[21:44:31 WARN]: java.lang.IllegalStateException: Failed to parse packetevents version!
[21:44:31 WARN]:        at HMCLeaves-1.1.0-beta.jar//io.github.fisher2911.hmcleaves.packetevents.spigot.util.updatechecker.UpdateChecker.checkLatestReleasedVersion(UpdateChecker.java:51)
[21:44:31 WARN]:        at HMCLeaves-1.1.0-beta.jar//io.github.fisher2911.hmcleaves.packetevents.spigot.util.updatechecker.UpdateChecker.checkForUpdate(UpdateChecker.java:62)
[21:44:31 WARN]:        at HMCLeaves-1.1.0-beta.jar//io.github.fisher2911.hmcleaves.packetevents.spigot.util.updatechecker.UpdateChecker.lambda$handleUpdateCheck$0(UpdateChecker.java:111)
[21:44:31 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
[21:44:31 WARN]: Caused by: java.io.IOException: Server returned HTTP response code: 403 for URL: https://api.github.com/repos/retrooper/packetevents/releases/latest
[21:44:31 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1997)
[21:44:31 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1589)
[21:44:31 WARN]:        at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224)
[21:44:31 WARN]:        at HMCLeaves-1.1.0-beta.jar//io.github.fisher2911.hmcleaves.packetevents.spigot.util.updatechecker.UpdateChecker.checkLatestReleasedVersion(UpdateChecker.java:45)
[21:44:31 WARN]:        ... 3 more
[21:44:31 INFO]: [packetevents] Something went wrong while checking for an update. Your build: (2.0.0)
[21:44:31 INFO]: [packetevents] [Checking for an update again in 20 seconds..

```
And console has been spammed by packetevents, this plugins is maybe integrated intro HMCLeaves

**Comment by Fisher2911** [comment]
I’m pretty sure this happens because of the update checker and if the server is restarted too many times, I’ll try to fix it by disabling the update checker for PacketEvents.

This shouldn’t cause any issues though (other than the spam) so you should be fine.

**Comment by Fisher2911** [comment]
This should be fixed in 1.3.0-beta.
